src/components/SearchBar/SearchBar.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { Box, TextField, IconButton } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+export const SearchContainer = styled(Box)(({ theme }) => ({
+  position: 'absolute',
```

Comment:
SearchContainer uses absolute positioning and many hard-coded pixel values and colors (position, width, height, left, top, background, backdropFilter, borderRadius). The project's CDD styling rules require use of theme.spacing(), theme.palette, theme.typography and avoid hardcoded px/hex values so components adapt to theme and are responsive.

**Sugerencia:**
```typescript
Rework layout to use relative, responsive layout (flexbox/grid) inside the parent rather than absolute screen coordinates. Replace hardcoded values with theme tokens, for example:
background: alpha(theme.palette.background.paper, 0.75),
borderRadius: theme.shape.borderRadius,
width: theme.spacing(40) // or responsive variants, etc.
Use theme.breakpoints to adapt sizes instead of fixed left/top coordinates.
```